emperor/support_files/js/controller.js

```diff
@@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
   // the area we have to present the plot is smaller than the total
   var plotWidth = this.$plotSpaceId.width();
 
+  // TODO: The below will need refactoring
+  // This is addressed in issue #405
   if (this.sceneViews.length === 1) {
     this.sceneViews[0].resize(0, 0, plotWidth, this.height);
   }
   else if (this.sceneViews.length === 2) {
-    this.sceneViews[0].resize(0, 0, 0.5 * plotWidth, this.height);
-    this.sceneViews[1].resize(0.5 * plotWidth, 0, 0.5 * plotWidth, this.height);
+    this.sceneViews[0].resize(0, 0, SCENE_VIEW_SCALE * plotWidth, this.height);
+    this.sceneViews[1].resize(SCENE_VIEW_SCALE * plotWidth, 0,
+                              SCENE_VIEW_SCALE * plotWidth, this.height);
   }
   else if (this.sceneViews.length === 3) {
-    this.sceneViews[0].resize(0, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[1].resize(0.5 * plotWidth, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[2].resize(0, 0.5 * this.height, plotWidth, 0.5 * this.height);
+    this.sceneViews[0].resize(0, 0,
+			      SCENE_VIEW_SCALE * plotWidth,
+			      SCENE_VIEW_SCALE * this.height);
+    this.sceneViews[1].resize(SCENE_VIEW_SCALE * plotWidth, 0,
+			      SCENE_VIEW_SCALE * plotWidth,
+			      SCENE_VIEW_SCALE * this.height);
+    this.sceneViews[2].resize(0, SCENE_VIEW_SCALE * this.height,
+			      plotWidth, SCENE_VIEW_SCALE * this.height);
   }
   else if (this.sceneViews.length === 4) {
-    this.sceneViews[0].resize(0, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[1].resize(0.5 * plotWidth, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[2].resize(0, 0.5 * this.height, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[3].resize(0.5 * plotWidth, 0.5 * this.height, 0.5 * plotWidth, 0.5 * this.height);
+    this.sceneViews[0].resize(0, 0, SCENE_VIEW_SCALE * plotWidth,
+			      SCENE_VIEW_SCALE * this.height);
```

Comment:
I just downloaded this file and it seems like you have a mixture of tabs and spaces, would you mind fixing that?